pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -1154,19 +1084,11 @@ func (b *Build) setupCosignVolumes(secrets []corev1.Secret) ([]corev1.Volume, []
 }
 
 var (
-	lowestSupportedPlatformVersion = semver.MustParse("0.3")
-
-	supportedPlatformAPIVersionsWithWindowsAndReportToml = []*semver.Version{semver.MustParse("0.9"), semver.MustParse("0.8"), semver.MustParse("0.7"), semver.MustParse("0.6"), semver.MustParse("0.5"), semver.MustParse("0.4")}
-	supportedPlatformAPIVersions                         = append(supportedPlatformAPIVersionsWithWindowsAndReportToml, semver.MustParse("0.3"))
+	supportedPlatformAPIVersions = []*semver.Version{semver.MustParse("0.11"), semver.MustParse("0.10"), semver.MustParse("0.9"), semver.MustParse("0.8"), semver.MustParse("0.7")}
```

Comment:
```suggestion
	supportedPlatformAPIVersions = []*semver.Version{semver.MustParse("0.9"), semver.MustParse("0.8"), semver.MustParse("0.7")}
```
This PR might be safer if we simply removed support for old APIs, and added support for new APIs in a separate PR (so that new behavior could be documented).

Reply:
See: https://buildpacks.io/docs/reference/spec/migration/platform-api-0.9-0.10/

Reply:
I'm good with that

Reply:
Updated in [47b04a7](https://github.com/buildpacks-community/kpack/pull/1305/commits/47b04a77877e6d55a8ebd44252802fb2100dac6b)